model/commitqueue/commit_queue.go

```diff
@@ -111,25 +110,26 @@ func (q *CommitQueue) Next() (CommitQueueItem, bool) {
 	return q.Queue[0], true
 }
 
-func (q *CommitQueue) Remove(issue string) (bool, error) {
+func (q *CommitQueue) Remove(issue string) (*CommitQueueItem, error) {
```

Comment:
Was there a reason for switching these to the items? If we don't end up using the item, I would argue against this refactor. Although we return nil, nil for all our DB functions (sigh) it's not really the best.